src/services/ExposureNotificationService/ExposureNotificationService.ts

```diff
@@ -144,7 +145,7 @@ export class ExposureNotificationService {
   }
 
   async updateExposureStatus(): Promise<ExposureStatus> {
-    if (this.exposureStatusUpdatePromise) return this.exposureStatusUpdatePromise;
+    if (!TEST_MODE && this.exposureStatusUpdatePromise) return this.exposureStatusUpdatePromise;
```

Comment:
Added this so that in test mode, I can force to refresh more efficiently.

Reply:
This is to make `updateExposureStatus` transactional (so that when invoked twice in a row there won't be 2 competing processes to fetch and match keys). How does it make efficient to test? it makes it broken, basically.